plugins/foreman_openscap/1.0/index.md

```diff
@@ -126,7 +126,9 @@ Edit ```/etc/foreman-proxy/settings.d/openscap.yml``` with the appropriate setti
 
 Workflow with Puppet and Ansible is similar. In the first step, you install appropriate package and import module or role which takes care of the client deployment. Foreman overrides parameters/variables for port, server and policies behind the scenes to make sure appropriate configuration is available to the client. When configuration is applied to host, foreman_scap_client is installed, client config is written to ```/etc/foreman_scap_client/config.yaml``` and a cron that will upload reports based on configuration of each policy is created.
 
-It is important to make sure *foreman_scap_client* package is available to your client system. Foreman plugins repo can be optionally enabled using the appropriate parameters/variables. The repo is disabled by default as users may prefer to use their own mirror.
+It is important to make sure the *foreman_scap_client* package is available on your client system. The package is included in the [Foreman client repository](https://yum.theforeman.org/client/3.13/el9/x86_64/).
+
+Do not enable the Foreman plugins repository on client systems. It is intended for Foreman server components. You can optionally enable the repo using the appropriate parameters/variables. It is disabled by default as users may prefer to use their own mirror.
```

Comment:
> Do not enable the Foreman plugins repository on client systems. It is intended for Foreman server components.

I'd omit this. I don't think anyone would enable it on their own unless explicitly told to do so.

Reply:
Dropped.